#! /bin/sh /usr/share/dpatch/dpatch-run
## 30_w_time.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Let fprintf print locale-friendly seconds #252575

@DPATCH@
diff -urNad procps-3.2.7~/w.c procps-3.2.7/w.c
--- procps-3.2.7~/w.c	2009-01-09 15:56:04.000000000 +1100
+++ procps-3.2.7/w.c	2009-01-09 16:14:56.000000000 +1100
@@ -83,7 +83,7 @@
     else if (t > 60)				/* > 1 minute */
 	fprintf(fout, " %2lu:%02u ", t/60, (unsigned) t%60);
     else
-	fprintf(fout, " %2lu.%02us", t, centi_sec);
+    fprintf(fout, " %2lu.%02us", t, centi_sec);
 }
 
 /**** stat the device file to get an idle time */
